ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -12,6 +12,10 @@
       ansible.builtin.include_vars:
         dir: "{{ cifmw_basedir }}/artifacts/parameters"
 
+    - name: Install certmanager operator role
+      ansible.builtin.import_role:
```

Comment:
Just a "formal"/"theoretical" point. Can this thing be considered part of the deployment? Or it's more like an infra step?

Reply:
@pablintino that's a good question. As per the infra playbook https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/02-infra.yml#L67 ci_nmstate role is added there. It make sense to add it in the infra setup itself. It is needed for both baremetal and control plane deployment.